parseAndPopulate/groupings.py

```diff
@@ -26,10 +26,9 @@
 import unicodedata
 import xml.etree.ElementTree as ET
 
+import utility.log as log
```

Comment:
Local imports should be grouped separately from third party imports, this should stay as is.

Reply:
Imports are sorted automatically on save - in the future we can add a setting that will specify how imports should be sorted.